src/ingest/SpecRepo.cpp

```diff
@@ -295,6 +304,28 @@ void SpecRepo::process(
       table, version, buildIdentityByTime(table->timeSpec), table->name, 0, SpecState::NEW, 0));
     return;
   }
+  //hack, if table has sql statement, switch to sql parser
```

Comment:
@shawncao I agree we should put this to another code path where we read from cmd or sqlscript in future.